mesh/src/main/java/no/nordicsemi/android/mesh/sensorutils/DeviceProperty.java

```diff
@@ -1,6 +1,7 @@
 package no.nordicsemi.android.mesh.sensorutils;
 
 import androidx.annotation.NonNull;
+
```

Comment:
Empty line?

Reply:
Android Studio default formatting inserts empty line, I have now removed it to not confuse :)